haiku-apps/nanodot/patches/nanodot-1.1.patchset

```diff
@@ -0,0 +1,51 @@
+From 94d33d8922c58738845d4ea2f7c63890d15429d1 Mon Sep 17 00:00:00 2001
+From: Shiroko <hhx.xxm@gmail.com>
+Date: Wed, 3 Jan 2018 22:58:40 +0000
+Subject: Add missing header 'PopUpMenu.h'
```

Comment:
You should try up streaming your changes: https://github.com/HaikuArchives/NanoDot/pulls